src/features/game/types/withdrawables.ts

```diff
@@ -1153,7 +1153,7 @@ export const BUMPKIN_WITHDRAWABLES: Record<
   "Dawn Lamp": () => true,
   "Eggplant Onesie": (state) =>
     canWithdrawBoostedWearable("Eggplant Onesie", state),
-  "Fox Hat": () => false, // Not Launched
+  "Fox Hat": () => canWithdrawTimebasedItem(new Date("2024-05-01")),
```

Comment:
```suggestion
  "Fox Hat": () => true,
```
Since the date already past should it be just `true`?